samples/bluetooth/peripheral_power_profiling/README.rst

```diff
@@ -0,0 +1,221 @@
+
+Configuration
+*************
+
+|config|
+
+The Peripheral Power Profiling sample allows configuring some of its options using Kconfig.
+You can set different configuration to check the related power consumption by your development kit.
+You can change the following options:
+
+   * :kconfig:option:`CONFIG_BT_POWER_PROFILING_DATA_LENGTH` - sets notification data length.
+   * :kconfig:option:`CONFIG_BT_POWER_PROFILING_NOTIFICATION_INTERVAL` - sets notification sending interval in milliseconds.
+      Notifications are started when the central device enables it by writing to the characteristic CCC descriptor.
+   * :kconfig:option:`CONFIG_BT_POWER_PROFILING_NOTIFICATION_TIMEOUT` - sets notification timeout.
+      Notification are sent until this timeout will be fired, after that the sample disconnects and goes to the system off mode.
+   * :kconfig:option:`CONFIG_BT_POWER_PROFILING_CONNECTABLE_ADV_DURATION` - sets connectable advertising duration in (N * 10) milliseconds unit.
+      If connection has not been established while advertising duration, the device will go to the system off state.
+   * :kconfig:option:`CONFIG_BT_POWER_PROFILING_NON_CONNECTABLE_ADV_DURATION` - sets non-connectable advertising duration in (N * 10) milliseconds unit.
+      After advertising end the device will go to the system off state if there is not any outgoing connection.
+   * :kconfig:option:`CONFIG_BT_POWER_PROFILING_NFC_ADV_DURATION` - sets advertising duration after the NFC field detection in (N * 10) milliseconds unit.
+      If connection has not been established while advertising duration, the device will go to the system off state.
+   * :kconfig:option:`CONFIG_BT_POWER_PROFILING_CONNECTABLE_ADV_INTERVAL_MIN` - sets connectable advertising minimum interval in (N * 0.625) milliseconds unit.
+   * :kconfig:option:`CONFIG_BT_POWER_PROFILING_CONNECTABLE_ADV_INTERVAL_MAX` - sets connectable advertising maximum interval in (N * 0.625) milliseconds unit.
+   * :kconfig:option:`CONFIG_BT_POWER_PROFILING_NON_CONNECTABLE_ADV_INTERVAL_MIN` - sets non-connectable advertising minimum interval in (N * 0.625) milliseconds unit.
+   * :kconfig:option:`CONFIG_BT_POWER_PROFILING_NON_CONNECTABLE_ADV_INTERVAL_MAX` - sets non-connectable advertising maximum interval in (N * 0.625) milliseconds unit.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/bluetooth/peripheral_power_profiling`
+
+.. include:: /includes/build_and_run.txt
+
+Testing
+=======
+
+This testing procedure assumes that you are using `nRF Connect for Mobile`_ or `nRF Connect for Desktop`_.
+After programming the sample to your development kit, you need additional device for measuring power consumption.
+We are recommending to use the `Power Profiler Kit II (PPK2)`_ for measuring.
+
+Testing with `nRF Connect for Desktop`_ and `Power Profiler Kit II (PPK2)`_
```

Comment:
```suggestion
Testing with `nRF Connect for Desktop`_ and `Power Profiler Kit II (PPK2)`_
```
I believe we should not use links in headings.

Reply:
Sure I will use it as plain text here